br/tests/lightning_duplicate_resolution_error_multiple_files/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright 2021 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+check_cluster_version 5 2 0 'duplicate detection' || exit 0
+
+mydir=$(dirname "${BASH_SOURCE[0]}")
+
+run_sql 'DROP TABLE IF EXISTS dup_resolve.a'
+run_sql 'DROP TABLE IF EXISTS lightning_task_info.conflict_error_v2'
+
+! run_lightning --backend local --config "${mydir}/config1.toml"
+[ $? -eq 0 ]
+
+tail -n 10 $TEST_DIR/lightning.log | grep "ERROR" | tail -n 1 | grep -Fq "[Lightning:Restore:ErrFoundDuplicateKey]found duplicate key"
+
+check_not_contains "the whole procedure completed" $TEST_DIR/lightning.log
+
+# import a second time
+
+run_sql 'DROP TABLE IF EXISTS dup_resolve.a'
+run_sql 'DROP TABLE IF EXISTS lightning_task_info.conflict_error_v2'
+
+! run_lightning --backend local --config "${mydir}/config2.toml"
```

Comment:
those 2 config file only diffs by `batch-size`, what's checked by import it again?

Reply:
I want to make sure that the `error` mode works as expected in the cases of different batch size. You can also check that for the `replace` mode, we also added test cases including different batch size.
https://github.com/pingcap/tidb/blob/464a12651bf09bb9f5cbb8e16d61a0a0df9f7803/br/tests/lightning_duplicate_resolution_replace_multiple_keys_clustered_pk/config.toml#L13
https://github.com/pingcap/tidb/blob/464a12651bf09bb9f5cbb8e16d61a0a0df9f7803/br/tests/lightning_duplicate_resolution_replace_one_key_multiple_conflicts_clustered_pk/config.toml#L13

Reply:
but batch-size = 1/2 doesn't make any difference as input file is large than that, so each file is in 1 engine

Reply:
Yes, in this case batch-size = 1/2 does not make any difference. It makes difference in cases like `lightning_duplicate_resolution_error`. Code modified.